google-http-client/src/main/java/com/google/api/client/util/DateTime.java

```diff
@@ -260,6 +263,11 @@ public int hashCode() {
    * NumberFormatException}. Also, in accordance with the RFC3339 standard, any number of
    * milliseconds digits is now allowed.
    *
+   * <p>Any time information beyond millisecond precision will be truncated. Prior to version 1.30.2
```

Comment:
will be --> is (tech writing lives in the eternal present)

Reply:
Changed.